db/glue.go

```diff
@@ -0,0 +1,48 @@
+package db
+
+import (
+	tmdb "github.com/tendermint/tm-db"
+)
+
+// FIXME: hack, remove
```

Comment:
???

Reply:
Sorry, I meant to include an explanatory comment here. This is transitional code to glue the new DB interface into places that expect `tm-db` objects - i.e. Multi-store and IAVL - since they will be refactored at a later stage in this implementation. I will make a clarification.